app/controllers/cookies_preferences_controller.rb

```diff
@@ -16,6 +16,10 @@ def create
     end
   end
 
+  def cookies_preference_set?
+    true
```

Comment:
This can't be correct, can it?

Reply:
This overrides the method in `ApplicationController` and makes the banner not show on this controller

Reply:
this makes the cookie banner never appear on the cookies page, to overide the general rule

Reply:
`Read the filename...`